src/app/Main/Body/QueryPage/Results/TableResults/TableContent-RowWithColSums.component.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsx jsx */
+import { css, jsx, SerializedStyles } from "@emotion/core";
+import TableCell from "@material-ui/core/TableCell";
+import _ from "lodash";
+import {
+    backgroundColor,
+    bfrPrimaryPalette,
+    primaryColor,
+} from "../../../../../Shared/Style/Style-MainTheme.component";
+
+const tableCellStyle = (isName: boolean): SerializedStyles => css`
+    box-sizing: border-box;
+    border-right: 1px solid lightgrey;
+    border-top: 1px solid ${primaryColor};
+    background-color: ${bfrPrimaryPalette[50]};
+    width: ${isName ? "160px" : "auto"};
+    min-width: ${isName ? "160px" : "auto"};
+`;
+
+const emptyCellStyle = css`
+    border-right: none;
+    border-bottom: none;
+    border-top: 1px solid ${primaryColor};
+    border-left: 1px solid ${primaryColor};
+    background-color: ${backgroundColor};
+`;
+
+function calculateColSum(
+    tableData: Record<string, string>[],
+    headerValue: string
+): number {
+    const colNumbers: number[] = tableData.map((tableRow) => {
+        const colNumber = tableRow[headerValue];
+        if (colNumber !== undefined) {
+            return Number.parseFloat(colNumber);

```

Comment:
Reject: I think you want to sum up counts, so parseInteger would be the right function to use.

Reply:
Done. Commit "Change parseFloat() to parseInt()"

Reply:
Undid it. I need parseFloat to display the relative values correctly.